docs/source/user_guide/using_datasets.rst

```diff
@@ -4446,14 +4454,43 @@ to a FiftyOne 3D scene is shown below:
 
 You can customize the appearance of a point cloud by setting the
 `default_material` attribute of the point cloud object, or dynamically from
-the app. Please refer to the :class:`fiftyone.core.threed.PointCloudMaterial`
+the app. Please refer to the
+:class:`PointCloudMaterial <fiftyone.core.threed.PointCloudMaterial>`
 class for more details.
 
-Initializing the point cloud with `flag_for_projection=True` allows 
-:func:`compute_orthographic_projection_images() <fiftyone.utils.utils3d.compute_orthographic_projection_images>`
-to generate
-:ref:`orthographic projection images <orthographic-projection-images>` of the
-point cloud.
+.. note::
+
+    Initializing a point cloud with `flag_for_projection=True` allows
+    :func:`compute_orthographic_projection_images() <fiftyone.utils.utils3d.compute_orthographic_projection_images>`
+    to generate
+    :ref:`orthographic projection images <orthographic-projection-images>` of
+    the point cloud.
```

Comment:
We can optionally mention that when this flag is absent, the first pointcloud is used to calculate ortho projection.

Reply:
Cool, I updated to clarify that `flag_for_projection=True` is only necessary if you want to select which point cloud to project